### DISCUSSION TITLE
Lots of confusion as to how things work

### DISCUSSION THREAD

**Discussion by simonmcnair** [beetbox/beets#4770] [Q&A] [answered]
I've been playing with beets for a while and I'm slowly getting to terms with the processes that make up the design.  Today thought, something weird happened.

I moved my fiels to a directory and tried to do a beets import.  It would not have it.  skipped everytime.  I tried changing the album metadata, no joy.  I assumed updating the ablum metadata would update the file modification time and trigger a fresh import, no joy.

So I relied on my fallback.  The also excellent, foobar2000.  I did the get tags from freedb and foobar updated them.  I then reran the beets import and it successfully reimported the same tracks perfectly happily.

Any idea why ?  I have chroma enabled.  I tried changing the tags, I updated the last modified but it still wouldn't find a release.

**Comment by arsaboo** [comment, marked as answer]
Without your config and seeing the logs, it is hard to say what is going on. Start simple, just have a basic config and use `-v` to see what is going in.
